tests/testthat/_snaps/conventional-commits/NEWS.md

```diff
@@ -0,0 +1,40 @@
+<!-- NEWS.md is maintained by https://cynkra.github.io/fledge, do not edit -->
+
+## upkeep
+ update rlang usage.
+
+## fix
+ prevent racing of requests
+
+Introduce a request id and a reference to latest request. Dismiss
+incoming responses other than from latest request.
+
+Remove timeouts which were used to mitigate the racing issue but are
+obsolete now.
+
+Reviewed-by: Z
+Refs: #123
+## feat(lang)
```

Comment:
we might want empty lines before and after h2?

Reply:
Yes, those would be great.